src/main/scala/uk/gov/hmrc/ct/computations/CP672a.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ */
+
+package uk.gov.hmrc.ct.computations
+
+import uk.gov.hmrc.ct.box._
+import uk.gov.hmrc.ct.computations.retriever.ComputationsBoxRetriever
+
+
+case class CP672a(value: Option[Int]) extends CtBoxIdentifier(name = "Out Of Proceeds from disposals from main pool")  with CtOptionalInteger with Input with SelfValidatableBox[ComputationsBoxRetriever, Option[Int]] {
+  override def validate(boxRetriever: ComputationsBoxRetriever) = {
+    collectErrors(
+      validateZeroOrPositiveInteger(),
+      validateNotMoreThan672Error(boxRetriever)
+    )
+  }
+
+  private def validateNotMoreThan672Error(retriever: ComputationsBoxRetriever) = {
```

Comment:
There's a validation function called belowMin that will do this for you

Reply:
@callum-gibbons updated with exceedMax function